### ISSUE TITLE
NFT Minting

### ISSUE THREAD

**Issue by kali6o4** [mit-dci/opencbdc-tx#72]
### Question

What's the logic behind minting NFTs on a cbdc?

Honestly just looking to dynamically recordkeep at a high throughput.

### Benefit

Potential business logic value

### Proposed Solution

I'll keep reading the documentation and relay any progress here (RTFM)

### Possible Difficulties

I have yet to quite understand whether minting is chain specific.

### Prior Work

I read alot.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by HalosGhost** [comment]
Though I don't think NFTs are necessarily out-of-scope for OpenCBDC to explore, I think this proposal needs more fleshing out (which you've already mentioned you are planning to do).

To begin, I think it might make sense to dig a little deeper into the question you would want answered. In particular, what do you mean “What's the logic behind…”? Do you mean “What changes would it take to…” or do you mean “What's the benefit of…” (or, indeed, something else).

**Comment by alibama** [comment]
https://ricardian.gitbook.io/ricardian-llc/ seems like a use case for nfts =

**Comment by HalosGhost** [comment]
@alibama use-cases aren't really the same thing as a research question. If you have an idea of how this proposal should be edited to make any of the sections more clear or actionable, that is welcome!

**Comment by alibama** [comment]
@HalosGhost 

i saw @kali6o4 say: 

Benefit
Potential business logic value

and followed that line of commentary to note some relevant code

i'm still trying to contextualize your work, frankly... it seems a bit in a bubble

**Comment by HalosGhost** [comment]
In some sense it definitely is in a bit of a bubble. This work is meant to help answer questions about potential designs, properties, and trade-offs of CBDCs (not necessarily to build one). That doesn't mean developments in other digital currency spaces are out-of-scope or irrelevant, but it does mean that the existence of something elsewhere doesn't necessarily mean it's useful or relevant here.

In short, for any proposal for research in this project, a clear view of a question to help answer is easily the most important aspect (which is why it's one of the only two required sections in the proposal form, and the first one presented).

NFTs are definitely not out-of-scope, but so far, what question could be answered by implementing them is quite unclear.
